types/react-instantsearch-dom/index.d.ts

```diff
@@ -55,21 +55,19 @@ interface CommonWidgetProps {
 }
 
 export interface InstantSearchProps {
-  apiKey: string;
-  appId: string;
```

Comment:
Those values aren't documented anymore but it's still possible to provide them. Note that they will be removed with the next major version of React InstantSearch. Until the release it's probably better to keep them.